changelogs/fragments/github_actions.yaml

```diff
@@ -0,0 +1,3 @@
+---
+trivial:
```

Comment:
> @jozzus I see you've added integration tests instead of unit test, plz add the unit tests for the scenario [here](https://github.com/ansible-collections/cisco.asa/blob/main/tests/unit/modules/network/asa/test_asa_ogs.py). We're not making bug fixes part of integration tests as running the integration tests is resource-heavy.

Hi @justjais thanks for correcting me, will check this.